tools/alpine/.detect-arch-var

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
please use `sh` not `bash`. Also having hidden files is kind of annoying, just use a normal one.